libflux/go/libflux/bench_test.go

```diff
@@ -106,24 +109,25 @@ func ParseAndDeserializeJSON(fluxFile string) error {
 func ParseAndReturnFB(fluxFile string) error {
 	p := libflux.Parse(fluxFile)
```

Comment:
I found that parsing in Rust takes a looong time, it may be because we do a lot of copying where we don't need to.  So I'm not that surprised that getting rid of copies does not improve things visibly.  Hopefully at some point that will change.